packages/orchestration/src/orchestration-api.ts

```diff
@@ -124,6 +122,24 @@ export interface Orchestrator {
     /** the Denom for the underlying asset on its issuer chain */
     baseDenom: Denom;
   };
+
+  /**
+   * Get denom on a holding chain.
+   *
+   * @param baseDenom denom on issuing chain
+   * @param base issuing chain
+   * @param chain holding chain - connection to base chain must be regitered in chainHub
+   * @returns denom on holding chain
+   */
+  getDenomOn: <
```

Comment:
new top level interface method will need @tribble review

Reply:
right...
+cc @mitdralla

Reply:
the name `getDenomOn` is sort of... uinspired

different arg names might help... `orch.getDenomOn(baseDenom, issuingChain, holdingChain)`

which suggests `orch.getHoldingChainDenom(baseDenom, issuingChain, holdingChain)`